src/common/types/card-validator.d.ts

```diff
@@ -0,0 +1,37 @@
+import 'card-validator';
+
+// Merge @types/card-validator with missing methods
```

Comment:
We need this because `@types/card-validator` is out of date and doesn't have the type definitions that match with the implementation.

Reply:
maybe add a `todo: remove this once @types/card-validator` is up to date with implementation?